content/20.pharmaceuticals.md

```diff
@@ -699,25 +699,45 @@ Dendritic cells are potent antigen-presenting immune cells that easily take up m
 Ultimately, primed T follicular helper cells can stimulate germinal center B cells that also present the viral antigen to produce antibodies against the virus [@doi:10.1016/j.immuni.2014.10.004].
 These cells are isolated from the patient, grown and transfected _ex vivo_, and reintroduced to the patient [@doi:10.1038/nrd.2017.243].
 
-mRNA-1273 was the first COVID-19 vaccine to enter a phase I clinical in the United States.
-ModernaTX, Inc. is currently spearheading an investigation on the immunogenicity and reactogenicity of mRNA-1273, a conventional lipid nanoparticle encapsulated RNA encoding a full-length prefusion stabilized spike (S) protein for SARS-CoV-2 [@clinicaltrials:NCT04283461].
-In a study that is ongoing, forty-five participants were enrolled and given intramuscular injections of mRNA-1273 in their deltoid muscle on day 1 and day 29, and then followed for the next twelve months.
+As of late 2020, promising results have been announced for several mRNA vaccines against SARS-CoV-2.
+mRNA-1273 was the first COVID-19 vaccine to enter a phase I clinical trial in the United States.
+In this trial, ModernaTX, Inc. spearheaded an investigation on the immunogenicity and reactogenicity of mRNA-1273, a conventional lipid nanoparticle encapsulated RNA encoding a full-length prefusion stabilized spike (S) protein for SARS-CoV-2 [@clinicaltrials:NCT04283461].
```

Comment:
Do you know what the difference is between how the pfizer vaccine works vs the moderna one? Is it the difference in how the mRNA is loaded into the cell?

Reply:
So, it seems like a lot of the info about the formulations isn't disclosed, but in the "RESULTS−MRNA VACCINES: MODERNA ANDBIONTECH/PFIZER" section of [this paper](https://pubs.acs.org/doi/pdf/10.1021/acsnano.0c07197) says that Moderna uses the prefusion conformation of the spike protein and Pfizer uses just the RBD, modified to reduce immunogenicity and increase translation.  They also speculate about some differences about what might be inside the liquid nanoparticles.

Reply:
I added some additional info about this in 8dfd8616!